tools/menuconfig.py

```diff
@@ -146,6 +150,8 @@ def touch_env():
         home_dir = os.environ['HOME']
     else:
         home_dir = os.environ['USERPROFILE']
+    
+    package_url = os.getenv('RTT_PACKAGE_URl') or DEFAULT_RTT_PACKAGE_URL
```

Comment:
`RTT_PACKAGE_URl` ? `RTT_PACKAGE_URL`

Reply:
I am not sure what do you mean. RTT_PACKAGE_URl is empty in your suggestion. The original line works fine as we want to get the envvar content of RTT_PACKAGE_URl. However, I just notices that "l" should be capital "L".